modules/specialized-guides/pages/large-deployments/hub-reporting.adoc

```diff
@@ -78,3 +78,37 @@ All times are in the local timezone of the server.
 If the peripheral servers are in different timezones, it is recommended to align all the schedules.
 Make sure that all reporting data are gathered at a specific point in time, and `update-reporting-hub-default` is running when all peripheral servers have actually finished their local jobs.
 
+=== Tuning Reporting Jobs
+
+[[reportdb-batch-size]]
+=== `report_db_batch_size`
+
+[cols="1,1"]
+|===
+| Description          | The maximum number of rows fetched and written from one database to the other in one shot.
+| Tune when            | Out of memory errors or on processing speed problems.
+| Value default        | 500
+| Value recommendation | 100 - 1000
+| Location             | [path]``/etc/rhn/rhn.conf``
+| Example              | `report_db_batch_size = 750`
+| After changing       | Check xref:memory-usage[memory usage].
+                         Monitor memory usage closely before and after the change.
+|===
+
+[[reportdb-hub-workers]]
+=== `report_db_hub_workers`
+
+[cols="1,1"]
+|===
+| Description          | The maximum number of workers requesting data from peripheral servers on a hub at the same point in time.
+| Tune when            | The number of peripheral servers increases significantly (more than 100), or a faster synchronization is required.
+| Value default        | 2
+| Value recommendation | 2 - 10
+| Location             | [path]``/etc/rhn/rhn.conf``
+| Example              | `report_db_hub_workers = 5`
+| After changing       | Check xref:memory-usage[memory usage].
+                         Monitor memory usage closely before and after the change.
```

Comment:
Maybe should we mention that the cpu usage during the synchronization should also be checked? If the cpu is already maxed out during the process they will not gain anything from increasing the number of workers. In fact in that case they might achieve better performance by lowering it.